mvp-spec/partials-updater/src/files.ts

```diff
@@ -3,12 +3,11 @@ import fs from "fs";
 import path from "path";
 
 const binDir = path.join(RootPath.path, "node_modules", ".bin");
-const assetsDir = path.join(RootPath.path, "..", "assets");
+export const assetsDir = path.join(RootPath.path, "..", "assets");
```

Comment:
I initially managed to avoid exporting constants from this file for a better encapsulation. Does it make sense to apply it to this variable?

Reply:
Thanks, it's a typo, I needed it at some point but not anymore. Will remove.